client/packages/programs/src/JsonForms/common/components/Array/Notes.tsx

```diff
@@ -124,7 +129,7 @@ const NotesComponent = (props: ArrayControlCustomProps) => {
     if (!child || typeof child !== 'object' || Array.isArray(child))
       return false;
     if (!enabled) return false;
-    if (!options?.['editRestrictions']) return true;
+    if (!options.editRestrictions) return true;
```

Comment:
Leftover from previous PR #2178, as per [this comment](https://github.com/openmsupply/open-msupply/pull/2178#discussion_r1316396702)